assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSegmentationSetupCTAWidget.test.js

```diff
@@ -122,6 +123,112 @@ describe( 'AudienceSegmentationSetupCTAWidget', () => {
 				queryByText( /Don’t show again/i )
 			).not.toBeInTheDocument();
 		} );
+
+		it( 'should render the widget when no audience is configured and Google Analytics data is loaded on the page', async () => {
+			const settings = {
+				configuredAudiences: [],
+				isAudienceSegmentationWidgetHidden: false,
+			};
+
+			// Set the data availability on page load to true.
+			registry
+				.dispatch( MODULES_ANALYTICS_4 )
+				.receiveIsDataAvailableOnLoad( true );
+
+			registry
+				.dispatch( MODULES_ANALYTICS_4 )
+				.receiveGetAudienceSettings( settings );
+
+			const { getByText, waitForRegistry } = render(
+				<AudienceSegmentationSetupCTAWidget Widget={ Widget } />,
+				{
+					registry,
+				}
+			);
+
+			// Wait for resolvers to finish to avoid an unhandled React state update.
+			await waitForRegistry();
+
+			expect(
+				getByText(
+					'Learn how different types of visitors interact with your site'
+				)
+			).toBeInTheDocument();
+
+			expect( getByText( 'Enable groups' ) ).toBeInTheDocument();
+		} );
+
+		it( 'should not render the widget when no audience is configured and Google Analytics data is not loaded on the page', async () => {
+			const settings = {
+				configuredAudiences: [],
+				isAudienceSegmentationWidgetHidden: false,
+			};
+
+			registry
+				.dispatch( MODULES_ANALYTICS_4 )
+				.receiveIsGatheringData( false );
+
+			// Set the data availability on page load to false.
+			registry
+				.dispatch( MODULES_ANALYTICS_4 )
+				.receiveIsDataAvailableOnLoad( false );
+
+			registry
+				.dispatch( MODULES_ANALYTICS_4 )
+				.receiveGetAudienceSettings( settings );
+
+			const { queryByText, waitForRegistry } = render(
+				<AudienceSegmentationSetupCTAWidget Widget={ Widget } />,
+				{
+					registry,
+				}
+			);
+
+			// Wait for resolvers to finish to avoid an unhandled React state update.
+			await waitForRegistry();
+
+			expect(
+				queryByText(
+					/Learn how different types of visitors interact with your site/i
+				)
+			).not.toBeInTheDocument();
+		} );
+
+		it( 'should not render the widget when configured audiences are present and Google Analytics data is loaded on the page', async () => {
+			const settings = {
+				configuredAudiences: [
+					audiencesFixture[ 0 ],
+					audiencesFixture[ 1 ],
+					audiencesFixture[ 2 ],
+				],
+				isAudienceSegmentationWidgetHidden: false,
+			};
+
+			// Set the data available on page load to true.
```

Comment:
Looks like this got left out. As this is a very minor change, I'll apply it myself. Thanks!